core/src/main/java/org/stellar/anchor/util/GsonUtils.java

```diff
@@ -0,0 +1,18 @@
+package org.stellar.anchor.util;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.time.Instant;
+
+public class GsonUtils {
+  private static Gson instance = null;
+
+  public static GsonBuilder builder() {
+    return new GsonBuilder().registerTypeAdapter(Instant.class, new InstantConverter());
+  }
+
+  public static Gson getGsonInstance() {
```

Comment:
In Java Singleton pattern, this is usually called `getInstance()`
https://www.baeldung.com/java-singleton

Reply:
https://github.com/stellar/java-stellar-anchor-sdk/pull/158/commits/130707dffdc2cc60462b7b10a8f0bc33f73fb739